en/docs/learn/configuring-oauth2-openid-connect.md

```diff
@@ -70,20 +71,73 @@ the users with an authorization server-based authentication.
-    | Enable HTTP basic auth for      | Selecting this option enables HTTP basic authentication to be used for client authentication, else client credentials will be included in the request body                                                                           | Selected                                   |
-    | client authentication           |                                                                                                                                                                                                                                      |                                            |
+    <div class="tg-wrap"><table>
+    <thead>
+    <tr>
+        <th>Field </th>
+        <th>Description</th>
+        <th>Sample Value</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Enable OAuth2/OpenIDConnect</td>
+        <td>Selecting this option enables OAuth2/OpenID Connect to be used as an authenticator for users provisioned to the Identity Server.</td>
+        <td>Selected</td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td>Selecting the Default check box signifies that the OAuth2/OpenID Connect credentials are the main/default form of authentication. <br>This removes the selection made for any other Default checkboxes for other authenticators.</td>
+        <td>Selected</td>
+    </tr>
+    <tr>
+        <td>Authorization Endpoint URL</td>
+        <td>This is a standard OAuth Authorization Endpoint URL of the federated IDP.</td>
+        <td><code>https://localhost:9443/oauth2/authorize/</code></td>
+    </tr>
+    <tr>
+        <td>Token Endpoint URL</td>
+        <td>This is a standard OAuth Token Endpoint URL of the federated IDP.</td>
+        <td><code>https://localhost:9443/oauth2/token/</code></td>
+    </tr>
+    <tr>
+        <td>Client Id</td>
+        <td>Client ID of the application you registered in the IDP for Identity Server.</td>
+        <td><code>1421263438188909</code></td>
+    </tr>
+    <tr>
+        <td>Client Secret</td>
+        <td>Client Secret of the application you registered in the IDP for Identity server. Click the Show button to view the value you enter.</td>
+        <td><code>12ffb4dfb2fed67a00846b42126991f8</code></td>
+    </tr>
+    <tr>
+        <td>Callback URL</td>
+        <td>This is the URL to which the browser should be redirected after the authentication is successful. It should be the <code>commonauth</code> endpoint of Identity server</td>
+        <td><code>https://localhost:9443/commonauth</code></td>
+    </tr>
+    <tr>
+        <td>OpenID Connect User ID Location</td>
+        <td>Select whether the User ID is found in the 'sub' attribute that is sent with the OpenID Connect request or if it is found among claims.</td>
```

Comment:
Selects?

Reply:
Here we are instructing the user to select an option, so using 'select' is appropriate right?